content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -115,15 +115,57 @@ OpenTelemetry's [Agent Configuration](https://opentelemetry.io/docs/zero-code/ja
 
 {{% /tab %}} -->
 
-<!-- {{% tab "Go" %}}
+{{% tab "Go" %}}
 
 ## Compatibility requirements
 
+The Datadog SDK for Go supports library instrumentations written using the [Opentelemetry-Go Trace API][1]. This includes the [opentelemetry-go-contrib/instrumentation][2] libraries, however, anything that relies on metrics or logs exporters is not supported at this time.
+
 ## Getting started
 
+Set up the Datadog trace provider and configure Opentelemetry to use the Datadog trace provider as per the Imports and Setup instructions listed in the [Go Custom Instrumentation using OpenTelemetry API guide][3].
+
+Then, follow the instructions of your chosen opentelemetry-go-contrib library to instrument your service.
```

Comment:
```suggestion
To set up the Datadog trace provider and configure Opentelemetry to use the Datadog trace provider, perform the following steps:

 1. Follow the steps in the Imports and Setup sections of the [Go Custom Instrumentation using OpenTelemetry API][3] page.
1. Follow the instructions for instrumenting your service with your chosen `opentelemetry-go-contrib` library.
```

Can you expand more on the second item above? Is it obvious where they find those instructions? Are those external instructions that don't involve Datadog stuff? Etc

Reply:
@neko-dd , 
Generally speaking, the customers reading this doc are familiar with opentelemetry integrations, and already have their hands on an integration that they'd like to integrate with ddtrace. Each integration will have its own setup instructions and I can't think up a generic link to paste here.
After the trace provider is configured, they follow Otel instructions (not Datadog). And there are no more Datadog components required.